packages/block-editor/src/components/inspector-controls-tabs/settings-tab.js

```diff
@@ -2,11 +2,13 @@
  * Internal dependencies
  */
 import AdvancedControls from './advanced-controls-panel';
+import PositionControls from './position-controls-panel';
 import { default as InspectorControls } from '../inspector-controls';
 
 const SettingsTab = ( { showAdvancedControls = false } ) => (
 	<>
 		<InspectorControls.Slot />
+		<PositionControls />
```

Comment:
This is an interesting one — I tried it like that originally, but we needed to have deliberate control over where the Position panel is output in the list of panels, and using `<InspectorControls>` as the parent means that the order is a bit harder to predict. By setting it here explicitly, we ensure that it always appears at the bottom, directly before the advanced controls, rather than using the order in which components are imported to determine the rendering order.